sources/packages/backend/libs/services/src/queue/queue.service.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable } from "@nestjs/common";
+import { InjectRepository } from "@nestjs/typeorm";
+import { QueueConfiguration } from "@sims/sims-db";
+import { Repository } from "typeorm";
+import { QueueModel } from "./model/queue.model";
+
+@Injectable()
+export class QueueService {
+  constructor(
+    @InjectRepository(QueueConfiguration)
+    private queueConfigurationRepo: Repository<QueueConfiguration>,
+  ) {}
+
+  /**
+   * Get all queue name and configurations.
+   * @returns queue configuration.
+   */
+  async getAllQueueConfigurations(): Promise<QueueConfiguration[]> {
```

Comment:
Is the query no cached anywhere?

Reply:
i added a comment, iam planning do the caching logic in my next PR